goon/browserassets/css/browserOutput.css

```diff
@@ -325,7 +325,7 @@ h1.alert, h2.alert			{color: #000000;}
 .sans					{font-family: 'Comic Sans MS', cursive, sans-serif;}
 .wingdings				{font-family: Wingdings, Webdings;}
 .robot					{font-family: 'PxPlus IBM MDA'; font-size: 1.15em;}
-.ancient				{color: #008B8B;	font-stye: italic;}
```

Comment:
So turns out this style has been broken for 4 years. huh.

Reply:
true to it's name